stack/.env.example

```
@@ -20,10 +20,16 @@ MUCGPT_SSO_CLIENTID=mucgpt
 MUCGPT_SSO_SCOPE=profile,openid,LHM,LHM_Extended
 MUCGPT_SSO_ROLE=lhm-ab-mucgpt-user
+# Redis
+MUCGPT_REDIS_HOST=localhost
+MUCGPT_REDIS_USERNAME=default
+MUCGPT_REDIS_PASSWORD=password
 # Langfuse
 MUCGPT_LANGFUSE_PUBLIC_KEY=
 MUCGPT_LANGFUSE_SECRET_KEY=
 MUCGPT_LANGFUSE_HOST=
+# MCP
+MUCGPT_MCP_SOURCES='{"example":{"url":"http://localhost:8086/mcp","forward_token":true}}'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)